js/libs/form.js

```diff
@@ -300,12 +303,17 @@ Fliplet().then(function () {
           }
 
           var $vm = this;
+          var entryLoaded = false;
+
+          if (typeof trackEvents === 'undefined') {
```

Comment:
We shouldn't use the presence of `trackEvents`, which is only used to dictate whether the reset should be tracked, to signify whether this "is a reset action".

The `isReset` flag also isn't entirely to future readers why this is necessary because the function is called `reset()`. We need better explanations and naming.

Reply:
@yogeshmsharma-xoriant I suggest refactoring to use functions like this:

```js
resetForm() {
},
submit() {
  // Submit the form as it does now

  // Reset form
  resetForm()
},
reset() {
  // Reset form
  resetForm()
}
```

This way, you can have code that runs in `reset()` that isn't in `resetForm()`. `reset()` is what happens when the reset is intentional whereas `resetForm()` is the internal function for resetting the form.